### ISSUE TITLE
homepage-blog picture/name of author

### ISSUE THREAD

**Issue by mikemccamon** [scorchingoyster/owasp-layout#29]
So for the article page we would have additional front matter

author: Harold Blankenship
author_title:
picture: /assets/harold.jpg
abstract: New blog post example content. Talk about using GitHub for the new website. More text to follow in a second here. Describe the functionality and the awesome CSS.

That way the article page has that data. Is there a way that the homepage-blog section could just pull that from "the most recent" _post and render it here? The abstract field would be the copy for home page. If author object is empty then we ignore the author-title and picture.

Crude layout design will be in Slack channel

![image_from_ios](https://user-images.githubusercontent.com/3586992/61175333-5a890380-a57b-11e9-9d9c-7677dcaa1fb0.jpg)

**Comment by scorchingoyster** [comment]
I thought it would be a pretty straightforward implementation, but after trying some things out and reading a lot of online community discussions—it seems that there isn't actually a good way to implement this without drawback.

surprisingly, "floating" the image to the bottom right while allowing the text to wrap around it isn't achievable without some hacky fix through javascript, and it's also limited by that the element won't be fluid and exact number of text lines will have to known so that the CSS/JS can account for it. 

in short, CSS doesn't seem to have a good way to support this layout behavior at the moment.

I suggests moving the photo and the author information to top right or left instead. if you do prefer to keep it bottom left, we may be able to use one of the workarounds, but it will mean that there will be some significant limitations that come with it.

(details here: https://stackoverflow.com/a/499883)

**Comment by mikemccamon** [comment]
Sorry you had to do so much research - top right works!


Mike McCamon | m: +1.913.940.2737 | Sent from iPhone

**Comment by scorchingoyster** [comment]
I actually do agree that the bottom right option looks better, it's what I often do with print layout as well. but CSS is another beast.

will do the top right option then.

**Comment by scorchingoyster** [comment]
your sketch seemed like you wanted to make use of the blue border that the news list, so I just replicated the effect for this:

![image](https://user-images.githubusercontent.com/3586992/61330904-aee7e980-a7ee-11e9-98ef-4aa84cc217ea.png)

without the photo and/or the author row, the text paragraph would just display as-is.

**Comment by mikemccamon** [comment]
Looks great!
